proposals/csharp-9.0/top-level-statements.md

```diff
@@ -67,7 +67,7 @@ they were combined in the block body of a `Main` method of a `Program` class in
 as follows:
 
 ``` c#
-static class Program
+internal partial class Program
```

Comment:
```suggestion
partial class Program
```
And the same for all the rest. This is what allows the user to choose their own accessibility.